styles.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@500;700&display=swap');
+
+* {
+  margin: 0;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  width: 100%;
+  align-items: center;
+}
+
+#my-logo {
+  font-size: 18px;
+  line-height: 24px;
+  padding-left: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+}
+
+.social-media a {
+  background-color: transparent;
+  border: none;
+}
+
+#my-logo > a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.menu-bar {
+  color: #acb7c3;
+  font-size: 36px;
+  padding-right: 24px;
+  margin-top: 19px;
+  margin-bottom: 19px;
+}
+
+.headline-section {
+  display: flex;
+  order: 0;
+  flex-direction: column;
+  background-color: #1c1a19;
+  width: 100%;
+  height: 650px;
+  background-image: url('/images/Group.png');
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.headline-section h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 56px;
+  margin: 120px 24px 12px;
+}
+
+.headline-section h2 {
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  margin: 0 24px;
+}
+
+#paragraph {
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  margin: 12px 24px;
+}
+
+.social-media {
+  margin: 0 24px;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```

DRY is a requirement, it is been marked optional because the previous code reviewer missed it, by mistake. We are humans.